Spoj1Divisor Summation/Program.cs

```diff
@@ -7,26 +7,49 @@
 
 namespace Spoj1Divisor_Summation
 {
-    public class Program
-    {
-        public static void Main()
-        {
-            var reader = new StreamReader(Console.OpenStandardInput());
-            var writer = new StreamWriter(Console.OpenStandardOutput());
+    public class Program
+    {
+        public static void Main()
+        {
+            var reader = new StreamReader(Console.OpenStandardInput());
+            var writer = new StreamWriter(Console.OpenStandardOutput());
             writer.AutoFlush = false;
-
-            DivisorSumation(reader, writer);
+            DivisorSumation(reader, writer);
+            Console.ReadLine();
         }
-
         public static void DivisorSumation(TextReader reader, TextWriter writer)
         {
-           //*******************************
-           //* TODO: Implement method body *
-           //*******************************
+            LiczbaPowtorzenObliczen(reader, writer);

```

Comment:
"LiczbaPowtorzenObliczen" nie do końca opisuje to co się dzieje w metodzie, można by dać nazwę "ObliczSumyPodzielników"